### ISSUE TITLE
Segmentation fault with `bun repl`

### ISSUE THREAD

**Issue by v1rtl** [oven-sh/bun#12132]
### How can we reproduce the crash?

- Install bun 1.1.16
- run `bun repl`
- observe the crash

Below I attached `adb bun repl` debugger session output

### Relevant log output

```shell
Starting program: /home/v1rtl/.bun/bin/bun repl
Downloading separate debug info for system-supplied DSO at 0x7ffff7fc7000                                      
[Thread debugging using libthread_db enabled]                                                                  
Using host libthread_db library "/usr/lib/libthread_db.so.1".
                                                                                                               
Program received signal SIGSEGV, Segmentation fault.
__memcpy_avx_unaligned_erms () at ../sysdeps/x86_64/multiarch/memmove-vec-unaligned-erms.S:366
Downloading source file /usr/src/debug/glibc/glibc/string/../sysdeps/x86_64/multiarch/memmove-vec-unaligned-erms.S
366             movq    %rsi, (%rdi)
```

### Stack Trace (bun.report)

Bun v1.1.16 ([`bf7b327`](<https://github.com/oven-sh/bun/tree/bf7b327f68568104a5e40c1944b7b255b02f0745>)) on linux x86_64 [BunxCommand]

Segmentation fault at address 0x5C824A75472F

- `??`
- [`semver.zig:268`](<https://github.com/oven-sh/bun/blob/bf7b327f68568104a5e40c1944b7b255b02f0745/src/install/semver.zig#L268>): `src.cli.bunx_command.BunxCommand.exec`
- [`StandaloneModuleGraph.zig:677`](<https://github.com/oven-sh/bun/blob/bf7b327f68568104a5e40c1944b7b255b02f0745/src/StandaloneModuleGraph.zig#L677>): `src.cli.Cli.start`
- [`array_list.zig:0`](<https://github.com/oven-sh/bun/blob/bf7b327f68568104a5e40c1944b7b255b02f0745/src/deps/zig/lib/std/array_list.zig#L0>): `process.argsAlloc`
- `??`
- `??`
- [`start.zig:487`](<https://github.com/oven-sh/bun/blob/bf7b327f68568104a5e40c1944b7b255b02f0745/src/deps/zig/lib/std/start.zig#L487>): `main`

**Comment by Jarred-Sumner** [comment]
Duplicate of #12070

**Comment by Jarred-Sumner** [comment]
Fixed in #12071, please run `bun upgrade --canary` to get the latest

**Comment by talentlessguy** [comment]
no longer crashes, thanks for fixing ❤️
